### ISSUE TITLE
Name authorships with code articles

### ISSUE THREAD

**Issue by mdoering** [CatalogueOfLife/data#250]
https://data.catalogueoflife.org/catalogue/3/names?q=contrary

**Comment by mdoering** [comment]
e.g. from Kew: https://data.catalogueoflife.org/dataset/1024/name/498191

**Comment by yroskov** [comment]
![image](https://user-images.githubusercontent.com/28574252/113191602-5c00d080-9223-11eb-84e4-8ca221335c90.png)

If GSD authors decided to add nomenclatural comments in taxon authorstrings, I do respect their choice. There are no problems, if CoL re-translates authorstrings as they present in the source. The same for authorstrings with comments like "sensu", "non", "pro parte", "nomen nudum", with use of square brackets and question marks - all of these has a big sense, and traditionally used in authorstrings.

**Comment by mdoering** [comment]
I would disagree. An authorstring is not the place to put any notes, it is called authorship for a reason. If its not the authorship it does not belong there. These nomenclatural notes should really be in Name remarks. Similar to sensu, non or pro parte being taxonomic remarks which we keep in Taxon/Synonym.phrase. Our name parsing does that for well known conventions like the taxonomic remarks you mentioned. But it cannot deal with just anything. This is highly unstructured data and im my eyes not suitable for entering a scrutinized COL. Can you imagine IPNI doing this? It is very valuable information for sure, but it just does not belong into the authorstring.

**Comment by yroskov** [comment]
CoL services may offer additional parsing, and deliver names, years and comments as itemized data outside the project, but presentation of authorstrings better to keep as they supplied by data providers.

**Comment by mdoering** [comment]
I would never want to see any of those in GBIF and would surely receive bad feedback on Twitter. And I doubt any other user of COL would be thrilled to have authors with long notes in them.

**Comment by redewalt** [comment]
I agree with Markus. Just authors included in authorstrings.
